README.md

```diff
@@ -157,7 +157,7 @@ provide the correct public-key.
 
 ### "What about this page that says DKIM can be fooled?"
 
-I lot of confident Twitter "experts" proclaim tha DKIM can be faked.
+A lot of confident Twitter "experts" proclaim tha DKIM can be faked.
```

Comment:
Nit: `tha` typo

Reply:
?

That's from another pull req fixing the typo: https://github.com/robertdavidgraham/hunter-dkim/pull/3